packages/table/src/cell/_common.scss

```diff
@@ -62,3 +62,9 @@ $cell-transition-duration: $pt-transition-duration * 3;
     cursor: $select-cell-cursor;
   }
 }
+
+@mixin cell-content-align-vertical() {
```

Comment:
That clears up naming confusion, and describes what it does: vertically center anything inside a cell. Not specific to loading or header cells. How's that @giladgray?